src/integrationTest/java/uk/gov/hmcts/dts/fact/mapit/MapitClientTest.java

```diff
@@ -84,4 +85,18 @@ void shouldReturnLocalAuthoritiyForPostcodeWhenMapitReturnsValue() {
         assertThat(localAuthority.isPresent()).isEqualTo(true);
         assertThat(localAuthority.get()).isEqualTo("Pembrokeshire Council");
     }
+
+    @Test
+    void shouldReturnAreaInformationForValidLocalAuthorityName() {
+        final Map<String, MapitArea> mapitAreaInfo =
+            mapitClient.getMapitDataForLocalAuthorities("Birmingham City Council");
+        assertThat(mapitAreaInfo.values().size()).isGreaterThan(0);
```

Comment:
AssertJ assertThat() provides a lot of useful comparison helper methods.  For this case, we could use assertThat(mapitAreaInfo.values()).isNotEmpty().

Just curious, why are you not asserting the map (mapitAreaInfo) directly is not empty, but assertion the values within the map?

Reply:
Checking mapitAreaInfo is not empty is sufficient here so I've updated the code as per your suggestion.